api/tests/rest.rs

```diff
@@ -95,7 +95,21 @@ fn test_start_api_tls() {
 	let server_addr = "0.0.0.0:14444";
 	let addr: SocketAddr = server_addr.parse().expect("unable to parse server address");
 	assert!(server.start(addr, router, Some(tls_conf)).is_ok());
```

Comment:
Why not sleep for a while if you think the server start need some time?

Reply:
Sure, was sure I wrote it, thanks!